common/notification/channels/inapp.ts

```diff
@@ -0,0 +1,19 @@
+import { Message, WebSocketService } from '../../websocket';
+import { AttachmentGroup } from '../../attachments';
+import { Channel, Context, Person, Notification } from '../types';
+import { User } from '../../user';
+
+export const inAppChannel: Channel = {
+    type: 'inapp',
+    async send(notification: Notification, to: User, context: Context, concreteID: number, attachments?: AttachmentGroup): Promise<void> {
+        const ws = WebSocketService.getInstance();
+        const message: Message = {
+            concreteNotificationId: concreteID,
+        };
+        await ws.sendMessageToUser(to.userID, message);
+    },
+    canSend: (_notification: Notification, user?: User) => {
```

Comment:
`user` here isn't really optional and we're anyways passing it to canSend in `notification/index.ts`. So maybe this and additionally the `Channel` interface needs to be adjusted.